### PR TITLE
F #src 2455 box arm home angles

### PR DESCRIPTION
**Pull request by ethanfowler**

### CODE REVIEW

**Review by beatrizleon — APPROVED** [review]

**Comment by ethanfowler** [comment]
@toliver I think @beatrizleon wanted your input on this!

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 1]
```diff
@@ -90,10 +91,10 @@
     <rosparam command="load" file="$(find sr_description)/hand/config/$(arg hand_id)_controller_gazebo.yaml"/>
     <include file="$(find ros_ethercat_model)/launch/joint_state_publisher.launch" />
     <!-- ARM -->
-    <rosparam if="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_trajectory_controller.yaml" command="load"/>
-    <rosparam unless="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_trajectory_controller_no_wrist.yaml" command="load"/>
+    <rosparam if="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_$(arg arm_controller_type)_controller.yaml" command="load"/>
+    <rosparam unless="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_$(arg arm_controller_type)_controller_no_wrist.yaml" command="load"/>
     <group if="$(arg arm_trajectory)">
```
Comment (toliver) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch:96]: Given that we are inside this `if arm_trajectory` I'm not sure that it makes sense to add an argument below to allow for another type of controller?
To be consistent we might have to change this `if`

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 2]
```diff
@@ -144,13 +145,13 @@
...
-      <rosparam unless="$(arg include_wrist_in_arm_controller)" command="load" file="$(find sr_robot_launch)/config/$(arg arm_id)_trajectory_controller_no_wrist.yaml"/>
+      <rosparam if="$(arg include_wrist_in_arm_controller)" command="load" file="$(find sr_robot_launch)/config/$(arg arm_id)_$(arg arm_controller_type)_controller.yaml"/>
+      <rosparam unless="$(arg include_wrist_in_arm_controller)" command="load" file="$(find sr_robot_launch)/config/$(arg arm_id)_$(arg arm_controller_type)_controller_no_wrist.yaml"/>
       <node name="arm_checker" pkg="sr_robot_launch" type="check_arms_ready" output="screen">
         <rosparam param="controllers" subst_value="True"> ["$(arg arm_id)_sr_ur_robot_hw"] </rosparam>
       </node>
-      <node name="arm_trajectory_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
-            args="--wait-for=/arms_ready --shutdown-timeout=1.0 $(arg arm_id)_trajectory_controller"/>
+      <node name="arm_$(arg arm_controller_type)_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
```
Comment (toliver) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: Same again for this node.

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_ur_arm_hand.launch (hunk 1 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch:96]: Fixed in 4529bc1e06372b03226014d39b6eb406063550b7.

File: sr_robot_launch/launch/sr_ur_arm_hand.launch (hunk 2 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch]: Fixed in 4529bc1e06372b03226014d39b6eb406063550b7.

**Comment by ethanfowler** [comment]
Resulting PR to sr_teleop_demos:
https://github.com/shadow-robot/sr_teleop_demos/pull/16

**Review by toliver — APPROVED** [review]

**Pull request merged**